packages/amplication-data-service-generator/src/server/create-server.ts

```diff
@@ -13,11 +14,24 @@ import { createPrismaSchemaModule } from "./prisma/create-prisma-schema-module";
 import { createGrantsModule } from "./create-grants";
 import { createDotEnvModule } from "./create-dotenv";
 import { createSeedModule } from "./seed/create-seed";
-import { BASE_DIRECTORY, SRC_DIRECTORY } from "./constants";
+import { BASE_DIRECTORY } from "./constants";
 import { createAuthModules } from "./auth/createAuth";
 
 const STATIC_DIRECTORY = path.resolve(__dirname, "static");
 
+const validatePath = (serverPath: string) => serverPath.trim() || null;
+
+const dynamicPathCreator = (serverPath: string) => {
```

Comment:
It could be much better if DynamicPathCreator would return a fixed type, and instead of sending the individual values for each function, we could pass the entire typed object and each function could use what ever it needs.